azurecontainerapps/azurecontainerapps.ts

```diff
@@ -138,11 +158,60 @@ export class azurecontainerapps {
                 console.log(tl.loc('DefaultResourceGroupMessage', resourceGroup));
             }
 
-            // Get the Container App environment if provided
-            const containerAppEnvironment: string = tl.getInput('containerAppEnvironment', false);
-            if (!util.isNullOrEmpty(containerAppEnvironment)) {
-                console.log(tl.loc('ContainerAppEnvironmentUsedMessage', containerAppEnvironment));
-                optionalCmdArgs.push(`--environment ${containerAppEnvironment}`);
+            // Set Container App environment deployment location, if provided
+            let location: string = tl.getInput('location', false);
+
+            // Ensure that the resource group that the Container App will be created in exists
+            const resourceGroupExists = await appHelper.doesResourceGroupExist(resourceGroup);
```

Comment:
I don't believe we need this `await`: the code in `doesResourceGroupExist` look synchronous.

Reply:
By the way, if there was a way to execAsync in this method, that could be nice as well, but I imagine you already considered that as a possibility.

Reply:
Removed the unnecessary `await`s that were in this file (I think they were autogenerated when I was using GitHub Copilot to complete some of these snippets).

As for the `execAsync` comment: unfortunately this command (or an equivalent) doesn't exist in the `azure-pipelines-task-lib/task` package from what I can tell, and I tried using the asynchronous `exec` command from the `child_process` library (since a couple other tasks are using it), but ran into a few issues with that during testing.